source/guide_codimd.rst

```diff
@@ -0,0 +1,303 @@
+.. highlight:: console
+.. author:: stunkymonkey <stunkymonkey.de>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/codimd.png
+      :align: center
+
+######
+CodiMD
+######
+
+CodiMD_ lets you create real-time collaborative markdown notes on all platforms.
```

Comment:
`CodiMD_` links to the guide itself instead of the project (https://github.com/hackmdio/codimd)